dotCMS/src/main/java/com/dotcms/contenttype/model/type/VanityUrlContentType.java

```diff
@@ -31,73 +31,73 @@
 @Gson.TypeAdapters
 @Value.Immutable
 public abstract class VanityUrlContentType extends ContentType implements Expireable,
-        Multilinguable {
+		Multilinguable {
 
-    private static final long serialVersionUID = 1L;
+	private static final long serialVersionUID = 1L;
 
-    private static final String TITLE_FIELD_NAME = "Title";
-    public static final String TITLE_FIELD_VAR = "title";
-    private static final String SITE_FIELD_NAME = "Site";
-    public static final String SITE_FIELD_VAR = "site";
-    private static final String URI_FIELD_NAME = "URI";
-    public static final String URI_FIELD_VAR = "uri";
-    private static final String FORWARD_TO_FIELD_NAME = "Forward To";
-    public static final String FORWARD_TO_FIELD_VAR = "forwardTo";
-    private static final String ACTION_FIELD_NAME = "Action";
-    public static final String ACTION_FIELD_VAR = "action";
-    private static final String ORDER_FIELD_NAME = "Order";
-    public static final String ORDER_FIELD_VAR = "order";
+	private static final String TITLE_FIELD_NAME = "Title";
+	public static final String TITLE_FIELD_VAR = "title";
+	private static final String SITE_FIELD_NAME = "Site";
+	public static final String SITE_FIELD_VAR = "site";
+	private static final String URI_FIELD_NAME = "URI";
+	public static final String URI_FIELD_VAR = "uri";
+	private static final String FORWARD_TO_FIELD_NAME = "Forward To";
+	public static final String FORWARD_TO_FIELD_VAR = "forwardTo";
+	private static final String ACTION_FIELD_NAME = "Action";
+	public static final String ACTION_FIELD_VAR = "action";
+	private static final String ORDER_FIELD_NAME = "Order";
+	public static final String ORDER_FIELD_VAR = "order";
 
-    public abstract static class Builder implements ContentTypeBuilder {
+	public abstract static class Builder implements ContentTypeBuilder {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this "Builder" class to an interface [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1694)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "Builder" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)